client.go

```diff
@@ -253,11 +253,14 @@ type Client struct {
 	nextPong          int64
 	lastSeen          int64
 	lastPing          int64
+	pingInterval      time.Duration
+	pongTimeout       time.Duration
 	eventHub          *clientEventHub
 	timer             *time.Timer
 	startWriterOnce   sync.Once
 	replyWithoutQueue bool
 	unusable          bool
+	overridePing      bool
```

Comment:
Actually I've meant a bit different - without the need to use `overridePing` flag. I've meant that at the moment of getting ConnectReply we can extract proper values for  pingInterval and pongTimeout - set them to Client - and then just use without calling transport method at all.